jenkins/build.sh

```diff
@@ -61,6 +61,7 @@ rpm -q rpm-build > /dev/null || {
 echo $VER > "$BASE_DIR"/VERSION
 /bin/chmod +rx "$BASE_DIR"/VERSION
 /bin/chmod +x "$BASE_DIR"/src/setup/rgw_setup
+#/bin/chmod +x "$BASE_DIR"/src/support/rgw_support_bundle
```

Comment:
Keeping this statement as commented for now.
as rgw_support_bundle already has +x permission and may not require to add again here.
Cc @rahul-tripathi-git